source/uwp/AdaptiveCardsNuget/AdaptiveCards.UWP.nuspec

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
-        <id>AdaptiveCards.UWP.Nuget.Beta</id>
-        <version>1.0.0</version>
+        <id>AdaptiveCards.Renderer.Uwp</id>
+        <version>1.0-beta.1</version>
```

Comment:
Change to 1.0.0-beta as we discussed https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages